rct229/ruletest_engine/ruletest_jsons/section21/rule_21_5.json

```diff
@@ -0,0 +1,792 @@
+                                                    {
+                                                        "id": "VAV Air Terminal 2",
+                                                        "is_supply_ducted": true,
+                                                        "heating_source": "HOT_WATER",
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilation_air_conditioning_system": "System 7"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilation_air_conditioning_systems": [
+                                            {
+                                                "id": "System 7",
+                                                "heating_system": {
+                                                    "id": "HHW Coil 1",
+                                                    "heating_system_type": "FLUID_LOOP",
+                                                    "hot_water_loop": "Boiler Loop 1"
+                                                },
+                                                "preheat_system": {
+                                                    "id": "Preheat Coil 1",
+                                                    "heating_system_type": "FLUID_LOOP",
+                                                    "hot_water_loop": "Boiler Loop 1"
+                                                },
+                                                "fan_system": {
+                                                    "id": "VAV Fan System 1",
+                                                    "fan_control": "VARIABLE_SPEED_DRIVE",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1"
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1"
+                                                        }
+                                                    ]
+                                                },
+                                                "cooling_system": {
+                                                    "id": "Cooling System 1",
+                                                    "rated_sensible_cool_capacity": 24619.0
+                                                }
+                                            }
```

Comment:
Since we are using `get_baseline_system_types` function in the rule definition, the system type 7 shall be a complete system type 7, otherwise, the rule definition will return not applicable results.
In this case, this test json is missing `cooling_system`, specifically, the chiller loop.

The `cooling_system` shall be add: 
"cooling_system_type": "FLUID_LOOP",
"chilled_water_loop": "Chiller Loop 1"

And at the RMI level, add chillers data
And inside the `fluid_loop`, add cooling loop data.